app/views/authors/_toc_node.html.haml

```diff
@@ -0,0 +1,15 @@
+%li
+  - if toc_node.is_a?(Manifestation)
+    - if toc_node.published?
+      = link_to toc_node.title, manifestation_path(toc_node)
```

Comment:
Here we need more nuance: if the node is an article in an issue of a periodical, or an item in an anthology by multiple authors, we want the name of the author(s) as well right here at the item level. We probably also want an indication (perhaps by icon) of the item's genre.

Precise UX for this screen is TBD, but I'm noting that this logic won't be enough.